pkg/cmd/importimages/importImages.go

```diff
@@ -0,0 +1,33 @@
+package importimages
+
+import (
+	run "github.com/rancher/k3d/cli"
+
+	"github.com/spf13/cobra"
+
+	"github.com/rancher/k3d/pkg/constants"
+)
+
+type options struct {
+	name     string
+	noRemove bool
+}
+
+func NewCommand() *cobra.Command {
+	// importImagesCmd represents the importImages command
+	opts := &options{}
+	var importImagesCmd = &cobra.Command{
+		Use:     "import-images [image] [image]...",
+		Short:   "Import list of container images from your local docker daemon into the cluster",
+		Aliases: []string{"import-images", "i"},
+		Args:    cobra.MinimumNArgs(1),
+		RunE: func(_ *cobra.Command, images []string) error {
+			return run.ImportImage(opts.name, opts.noRemove, images)
+		},
+	}
+
+	importImagesCmd.Flags().StringVarP(&opts.name, "name", "n", constants.DefaultK3sClusterName, "import images into a cluster")
+	importImagesCmd.Flags().BoolVarP(&opts.noRemove, "no-remove", "k", false, "Disable automatic removal of the tarball")
```

Comment:
Not sure, if the selection of `no-remove` and `k` is the best choice here since we had `no-remove`, `no-rm`, `keep` and `k` before. In any case, we'll need a deprecation notice, if we're going to drop some of them.